maven project/buy-a-car/src/main/java/com/retail/shop/car/dto/CarDto.java

```diff
@@ -0,0 +1,19 @@
+package com.retail.shop.car.dto;
+
+public class CarDto {
+    private String name;
+    private String engine;
```

Comment:
recommend Engine to be type of its own rather than a String - Will be more easy to infer

Reply:
actually I didn't know what to put here, just made something up, I will update when I extend the UI layer